gammapy/region/reflected.py

```diff
@@ -0,0 +1,68 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+from . import RegionList, PixCircleRegion
+
+__all__ = [
+    'find_reflected_regions',
+]
+
+
+def find_reflected_regions(region, center, exclusion_mask,
+                           angle_increment=0.1, min_distance=0):
+    """Find reflected regions.
+
+    Converts to pixel coordinates internally
+
+    Parameters
+    ----------
+    region : `~gammapy.region.Region`
+        Region
+    center : `~astropy.coordinates.SkyCoord`
+        Rotation point
+    exclusion_mask : `~gammapy.region.ExclusionMask`
+        Exlusion mask
+    angle_increment : float
```

Comment:
Even if it's a bit more typing, I think we should require `Angle` objects as inputs and not use float with assumed unit "deg".